### PR TITLE
Fix memory leak when reloading workspace

### PR DESCRIPTION
**Pull request by jnhyperion**

### CODE REVIEW

**Comment by jnhyperion** [comment]
https://github.com/robotframework/RIDE/issues/1868

**Comment by HelioGuilherme66** [comment]
@jnhyperion @Nyral With this PR, the Grid Editor will show the About content (which used to be temporary). Whenever a user opens a file, they must select an item on Tree. This may be an additional step, and is a noticeable change. Don't you think we should have this configurable?

**Comment by jnhyperion** [comment]
Or we can make some more changes to prevent tree nodes will be created twice when reloading workspace, this is only a minimum and "lazy" fix. what do you think?

**Comment by HelioGuilherme66** [comment]
Yes, I would prefer to keep root node selected. Why are they being duplicated?

**Comment by jnhyperion** [comment]
it's because message RideDataFileSet is published right after tree nodes are created, and the root node is selected, then the message handlers remove all the nodes and recreate new nodes. this behaviour causes memory leak. I don't know why, maybe it's a wxpython bug, but unfortunately I cannot manage to prove it.

File: utest/ui/test_tree.py [hunk 1]
```diff
@@ -186,7 +187,7 @@ def test_adding_test(self):
 class TestNodeSearchAndSelection(_BaseSuiteTreeTest):
 
     def test_topsuite_node_should_be_selected_by_default(self):
-        assert_equal(self._get_selected_label(), 'Top Suite')
+        assert_equal(self._get_selected_label(), '')
```
[the remark this answers is not in the document]
Comment (jnhyperion) [reply utest/ui/test_tree.py]: reverted

File: src/robotide/ui/mainframe.py [hunk 2]
```diff
@@ -474,6 +474,12 @@ def open_suite(self, path):
                 # raise err  # Just leave message in Parser Log
                 return False
         self._populate_tree()
+        import gc
+        gc.collect()
+        import objgraph
```
[the remark this answers is not in the document]
Comment (HelioGuilherme66) [reply src/robotide/ui/mainframe.py]: This does not look like essential code.

**Review by HelioGuilherme66 — CHANGES_REQUESTED** [review]
Did you forgot to push the changes?

File: src/robotide/ui/mainframe.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jnhyperion) [reply src/robotide/ui/mainframe.py]: already reverted this part, see https://github.com/robotframework/RIDE/pull/2276/files "File Changed" directly.

**Review by HelioGuilherme66 — APPROVED** [review]

**Pull request merged**